datatools/env.yml

```diff
@@ -1,13 +1,5 @@
-# AUTH0_CLIENT_ID: your-auth0-client-id
-# AUTH0_DOMAIN: your-auth0-domain
-# AUTH0_SECRET: your-auth0-secret
-# AUTH0_TOKEN: your-auth0-token
 DISABLE_AUTH: true
-OSM_VEX: http://localhost:1000
-SPARKPOST_KEY: your-sparkpost-key
-SPARKPOST_EMAIL: email@example.com
-GTFS_DATABASE_URL: jdbc:postgresql://postgres/catalogue
-GTFS_DATABASE_USER: postgres
-GTFS_DATABASE_PASSWORD: postgres_password
-MONGO_URI: mongodb://mongo
```

Comment:
this will berak the docker-compose.

I think it is ok, so you can continue to work (nobody is using compose right now) and we’ll fix it later.